lib/cli/src/generators/SVELTE/index.ts

```diff
@@ -1,9 +1,47 @@
+import fse from 'fs-extra';
+import { logger } from '@storybook/node-logger';
+
 import { baseGenerator, Generator } from '../baseGenerator';
 
 const generator: Generator = async (packageManager, npmOptions, options) => {
-  baseGenerator(packageManager, npmOptions, options, 'svelte', {
+  await baseGenerator(packageManager, npmOptions, options, 'svelte', {
     extraPackages: ['svelte', 'svelte-loader'],
+    extraAddons: ['@storybook/addon-svelte-csf'],
   });
+
+  let conf = fse.readFileSync('./.storybook/main.js').toString();
+
+  // add *.stories.svelte
+  conf = conf.replace(/js\|jsx/g, 'js|jsx|svelte');
```

Comment:
This means we can break Svelte CLI while touching `lib\cli\src\generators\configure.ts` Do you think it's best to pass an option "extension" that defaults to js, jsx, ts, txs ?

Reply:
My first implementation was checking if "framework is svelte" (there is already a special case for angular in this file), my 2nd implementation was doing that (a special properties for extensions), and my third is this PR :)